setup.sh

```diff
@@ -7,6 +7,7 @@ apt-get install -y python-scapy
 apt-get install -y tcpdump
 apt-get install -y nmap
 apt-get install -y python-urllib3
+apt-get install -y cme
```

Comment:
That's not working:
Package cme in raspbian sources (https://packages.debian.org/stretch/cme) is not what we're looking for...
In this context cme means CrackMapExec (https://github.com/byt3bl33d3r/CrackMapExec/wiki/Installation) which can be installed like so:
`#~ apt-get install -y libssl-dev libffi-dev python-dev build-essential
#~ sudo pip install crackmapexec`

Reply:
Where did you spot this urllib3 errors?

Right now I'm testing the original code (in raspbian stretch) and I'm curious if your changes are necessary.